### PR TITLE
sg: improve `sg` web testsuites

### PR DESCRIPTION
**Pull request by valerybugakov**
## Context

1. Make `sg test web-*` commands more consistent.
2. Improve respective docs.

## Test plan

Ensure that all `sg test web-*` commands work properly.

1. `sg test web-e2e`
2. `sg test web-regression`
3. `sg test web-integration`
4. `sg test web-integration:debug`

### CODE REVIEW

**Review by burmudar — APPROVED** [review]
Such a nice quality of life change this! Thanks! 

I still want to add a note to these to match them up with the CI pipeline steps. Ie. This command corresponds to Job X on the pipeline.

**Review by umpox — APPROVED** [review]
Nice!

**Review by jhchabran — APPROVED** [review]
Thank you for this!

File: doc/dev/background-information/sg/reference.md [hunk 1]
```diff
@@ -310,17 +311,18 @@ Available testsuites in `sg.config.yaml`:
 * bext-e2e
 * bext-integration
 * client
-* client-e2e
-* client-regression
 * docsite
+* web-e2e
 * web-integration
+* web-integration:debug
+* web-regression
```
Comment (limitedmage) [inline doc/dev/background-information/sg/reference.md:318]: For consistency, can we add `sg test web-unit` for Jest unit tests?

**Pull request merged**